stanford/roles/cluster/defaults/main.yml

```diff
@@ -155,10 +155,17 @@ CLUSTER_SECURITY_GROUP_RULES:
       to_port: 22
       cidr_ip: 10.0.0.0/16
   rabbitmq:
+    # See 'Firewalled nodes' section at
+    # http://www.rabbitmq.com/clustering.html
+    # 65500 - 65510
```

Comment:
I had originally opened this port range while debugging, but I believe it can now be removed.